src/main/java/edu/kis/vh/nursery/DefaultCountingOutRhymer.java

```diff
@@ -0,0 +1,40 @@
+package edu.kis.vh.nursery;
+
+public class DefaultCountingOutRhymer {
+
+	private static final int COUNT_WHEN_FULL = 11;
+
+	private static final int DEFAULT_RETURN_VALUE = -1;
+
+	private static final int MAX_NUMBERS_COUNT = 12;
+
+	private int[] numbers = new int[MAX_NUMBERS_COUNT];
+
+	private int total = DEFAULT_RETURN_VALUE;
```

Comment:
Errors in collaborators work (lack of good review). Extracting constants is done incorrectly. Have you asked for review hints (1.4.w7): tiny.cc/998muz ?
This constant should be different (separate) than the one in peekaboo and countOut methods.